src/cli.rs

```diff
@@ -13,19 +13,26 @@ pub struct Args {
     pub target: Option<Target>,
     pub target_dir: Option<PathBuf>,
     pub docker_in_docker: bool,
+    pub project_dir: Option<PathBuf>,
 }
 
 pub fn parse(target_list: &TargetList) -> Args {
     let mut channel = None;
     let mut target = None;
+    let mut project_dir: Option<PathBuf> = None;
     let mut target_dir = None;
     let mut sc = None;
     let mut all: Vec<String> = Vec::new();
-
+    
     {
         let mut args = env::args().skip(1);
         while let Some(arg) = args.next() {
-            if let ("+", ch) = arg.split_at(1) {
+            if arg == "--manifest-path" {
+                all.push(arg);
+                let path = args.next().expect("Missing argument in --manifest-path");
```

Comment:
Use `if let Some(..)` for this and also add the case for `--manifest-path=`.

Reply:
> > This command don't work:
> > `cross build --manifest-path clients/Cargo.toml --target armv7-linux-androideabi --release`
> 
> Can you post the output of the failed command?

It is fallback to host cargo and fail in linking

Reply:
I don't understand :-( I am newbies in rust, please help me.

Reply:
Look at the code below, there it is used in the same way.

Reply:
```rust
if let Some(path) = args.next() {
```

Reply:
ok, fix